recipes/perl-grass/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: perl-grass
+  version: '1.1.6'
+
+build:
+  number: 2
+  skip: True # [osx]
+
+source:
+  fn: v1.1.6.tar.gz
+  url: https://github.com/cancerit/grass/archive/v1.1.6.tar.gz
+  md5: 81acae11f5687d070166cda26f069de3
+
+requirements:
+  build:
+    - perl-threaded
+    - perl-app-cpanminus
+    - perl-file-sharedir-install
+    - perl-module-build
+    - perl-bio-db-sam
+    - perl-bioperl
+    - perl-gd
+    - perl-ipc-system-simple
+    - perl-sanger-cgp-vcf
+    - perl-encode-locale
+    - perl-lwp-simple
+    - perl-xml-parser
+  run:
+    - perl-threaded
+    - perl-ipc-system-simple
+    - perl-bioperl
+    - perl-bio-db-sam
+    - perl-bio-samtools
+    - perl-gd
+    - perl-sanger-cgp-vcf
+
+#test:
+#  commands:
```

Comment:
It there any reason to not include this test? Would be great to have a test in place for all bioconda recipes.